SessionMessagingKit/Sending & Receiving/Message Handling/MessageReceiver+ExpirationTimers.swift

```diff
@@ -100,14 +100,5 @@ extension MessageReceiver {
                 db,
                 Contact.Columns.lastKnownClientVersion.set(to: version)
             )
-        
```

Comment:
Just out of curious, are we sure we can remove this banner? Is there any place that we can see what the analytics of versions of Session iOS is distributed? @mpretty-cyro

Reply:
Yep just confirmed that we are happy to remove it (it's been 3 months since disappearing messages was released so _should_ have been out long enough for anyone that was planning on updating to install the update)